docs/debugging-install.md

```diff
@@ -0,0 +1,59 @@
+# Debugging Installation Issues
+
+## General
+
+#### `invalid configuration: no configuration has been provided`
```

Comment:
Bump all these up one level (3 #s, not 4) (wouldn't nit like this but I've got a couple of more substantial comments)